package/MDAnalysis/visualization/streamlines.py

```diff
@@ -172,7 +174,7 @@ def produce_list_indices_point_in_polygon_this_frame(vertex_coord_list):
             list_indices_point_in_polygon.append(index_list_in_polygon)
         return list_indices_point_in_polygon
 
-    def produce_list_centroids_this_frame(list_indices_in_polygon):
+    def produce_list_centroids_this_frame(list_indices_in_polygon: List[int]) -> List[float]:
```

Comment:
`Sequence[int]` instead of `List[int]` in the args, for same reason, maybe?

Also, for the return value, I see a comment indicating that `list_centroids_this_frame` may contain arrays instead of just floats? And looking at the control flow, `list_centroids_this_frame` could also contain the string `empty`? So, maybe a bit more complex than `List[float]`?

Reply:
Sequence[Optional[float]] but Optional gives None so i dont know  if it will return empty string